src/App.js

```diff
@@ -0,0 +1,85 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+import { FiTrash, FiPlus } from "react-icons/fi";
+
+function App() {
+
+  const [todos, setTodos] = useState(JSON.parse(localStorage.getItem("todos")) || []);
```

Comment:
To improve perfomance use lazy state initialization
```suggestion
  const [todos, setTodos] = useState(() => JSON.parse(localStorage.getItem("todos")) || []);
```